go.mod

```diff
@@ -15,11 +15,14 @@ require (
 	github.com/go-playground/universal-translator v0.16.0 // indirect
 	github.com/golang-migrate/migrate v3.5.4+incompatible // indirect
 	github.com/golang/mock v1.3.1
+	github.com/goware/modvendor v0.0.0-20190516042800-ce72b408a8fe // indirect
 	github.com/influxdata/influxdb v1.7.6 // indirect
 	github.com/jroimartin/gocui v0.4.0
+	github.com/jteeuwen/go-bindata v3.0.7+incompatible // indirect
```

Comment:
how go-bindata ended up in go.mod file? i don't think that app imports it. generated file for migrations isn't importing it.

Reply:
`make install-dev` does `go get -u github.com/jteeuwen/go-bindata` and apparently it adds it here. I added `go mod tidy` in `make install-dev` which removes unneeded modules and it's not present now.